docs/Reference/Rest_API/Rest.md

```diff
@@ -4,11 +4,44 @@ description: Teku RESTful APIa
 
 # REST API
 
+[Enable the REST API](#enable-the-rest-api) from the command line, and enable the API documentation endpoint
+to interact with the APIs using Swagger UI.
+
+The [Teku REST API documentation] is also available to view information about the REST APIs.
+
+## Enable the REST API
+
+Enable the REST API service by setting the [`--rest-api-enabled`](../CLI/CLI-Syntax.md#rest-api-enabled)
+command line option to `true`.
+
+You can enable the Swagger UI endpoint to visually interact with the APIs by setting the
```

Comment:
visually doesn't seem like the right word. interact using OpenAPI framework? Or do we need to explain - normal interaction can be done via curl but we also have this other way of doing it via the web browser

Reply:
Done